packages/katello/katello-installer-base/katello-installer-base.spec

```diff
@@ -118,6 +118,10 @@ ln -sf %{_datadir}/foreman-installer-katello/bin/katello-certs-check %{buildroot
 %doc README.*
 
 %changelog
+* Tue Oct 23 2018 Adam Price <komidore64@gmail.com> - 3.10.0-0.2.master
```

Comment:
It fails the check since the old release its comparing in git against is 3.10.0-1. Means we either just accept this one off and ignore it, or you alter the release on the old changelog commit to 3.10.0-0.1. I can live with either. Might feel weird to alter that release but its a nightly package.

Reply:
@ehelms I tried setting the previous changelog entry to `3.10.0-0.1.master` but the comparison job still fails in the same place.

Reply:
It fails because it does a `git checkout rpm/develop`, calculates the version there and then `git checkout $PR`. It compares those 2. It's a very good check, but in this case we should either accept it and merge it while we purge it from koji and repos or wait till 3.11.0.